pyproject.toml

```diff
@@ -58,6 +58,7 @@ test = [
     'rundoc>=0.4.3,<0.5',
     'pytest-subtests>=0.5,<1.0',
     'pytest-runner >= 2.11.1',
+    'pyarrow',
```

Comment:
We should set a minimum version

Reply:
Also can we move it as an `extra` requirement and not `test`.
```
[project.optional-dependencies]
copulas = ['copulas>=0.11.0',]
pyarrow = ['pyarrow>=version']
test = [
    'rdt[pyarrow]',
    'rdt[copulas]',

    'pytest>=3.4.2',
    'pytest-cov>=2.6.0',
    'jupyter>=1.0.0,<2',
    'rundoc>=0.4.3,<0.5',
    'pytest-subtests>=0.5,<1.0',
    'pytest-runner >= 2.11.1',
    'pyarrow',
    'tomli>=2.0.0,<3',
]
```